### ISSUE TITLE
Consider deploying the visualizer in GitHub pages

### ISSUE THREAD

**Issue by Peque** [dedukun/micromouse-webgl#2]
Or somewhere else, to show off and to make it easier for anybody to see how it looks like without the need to download the repository. :innocent:

**Comment by dedukun** [comment]
Yeah, that would probably be for the best, but I'm thinking of first resolving some of the performance issues, as well as #3, before doing the first release and having the visualizer hosted somewhere.

For now, I think I'll just do the README, and have some screenshots in it.

**Comment by Peque** [comment]
@dedukun Awesome! Good luck! :blush: :muscle:
